packages/engine/src/interaction/components/InteractableComponent.ts

```diff
@@ -319,6 +321,16 @@ export const InteractableComponent = defineComponent({
       }
     }, [isEditing.value])
 
+    useEffect(() => {
```

Comment:
Oh I'm not thinking straight.

XRUI already has its own API for consuming DOM events. This solution is the wrong level of abstraction. I will get back to you soon with pointers as to the correct solution.